Monal/Classes/MLConstants.h

```diff
@@ -164,6 +164,7 @@ static inline NSString* _Nonnull LocalizationNotNeeded(NSString* _Nonnull s)
 #define kMLResourceBoundNotice @"kMLResourceBoundNotice"
 #define kMonalFinishedCatchup @"kMonalFinishedCatchup"
 #define kMonalFinishedOmemoBundleFetch @"kMonalFinishedOmemoBundleFetch"
+#define kMonalKnownDevicesUpdated @"kMonalKnownDevicesUpdated"
```

Comment:
since this is used for every omemo update, maybe use this name

```suggestion
#define kMonalOmemoStateUpdated @"kMonalOmemoStateUpdated"
```